docs/using-connect/high-level.mdx

```diff
@@ -1,40 +1,41 @@
 ---
-description: 🏖️ Accessing Connect Prices
-title: 🏖️ Accessing Connect Prices
+description: Getting prices from the Connect oracle
+title: Getting Prices
+icon: tag
 ---
 
 <Note>
-    **Building with Connect? First step is to join our [Discord](https://discord.gg/amAgf9Z39w)**
+    **Building with Connect? Join our [Discord](https://discord.gg/amAgf9Z39w)**!
 </Note>
 
 ## Summary
 
-Connect prices are stored within the [`x/oracle`](https://github.com/skip-mev/slinky/tree/main/x/oracle) module by default.
+Connect prices are stored within the [x/oracle](https://github.com/skip-mev/slinky/tree/main/x/oracle) module.
 
 On very specific chains - right now only dYdX - they live in a different storage module (in dYdX's case, `x/prices`).
```

Comment:
Do we even want to include this as a callout?  I think we should just tell people "the way to use slinky is to use `x/oracle`" and not complicate things

Reply:
imo its fine as is, given we have two modules oracle/marketmap. i removed the `by default` to affirm what you're saying